src/main/java/com/squareup/kotlinpoet/AnnotationSpec.kt

```diff
@@ -20,6 +20,7 @@ import java.io.IOException
 import java.io.StringWriter
 import java.lang.reflect.Array
 import java.util.Arrays
+import java.util.Locale
```

Comment:
I don't want to add any new dependencies on the JVM. An easy way around this is to add the raw output string to each enum value.